### PR TITLE
add draft for windsurf-life-strategy

### PR DESCRIPTION
**Pull request by sstiefel19**

### CODE REVIEW

File: windsurf-life-strategy.md
Comment (KathSupp) [inline windsurf-life-strategy.md:1]: Line 25 to 48: not sure if a second and third metaphor is necesarry. I would stick with the windsurfing-metaphor

File: windsurf-life-strategy.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+You are sad, life is treating you like shit, and you blame specifically yourself for it, you might say something like. Yeah well, I don’t have a strategy, I just do what I want and run away every time I hate Heidelberg. No wonder there is no success in my life and other people don’t recognize and appreciate me. 
+your label for your self-perceived strategy (SPS): not existent
+
+scenario 2)
+You are in Tarifa, you feel great and realize there are so many great, appreciative people on this planet, that you don’t worry at all about anything.
+your label for your SPS:  Enjoy life to its fullest, trying to do sth good for the world at the same time
+
+SPS1 and SPS2 are very different, but O’s ES’s label will be the same in both scenarios. A physicist might have the idea to say: 
+ESs transform invariantly whereas SPS’s transform variantly under certain transformations in Minkowski space.
+Or, another anology from phyics:
```
Comment (KathSupp) [inline windsurf-life-strategy.md:67]: Not sure if this third metaphor is necessary.

File: windsurf-life-strategy.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+Anyway, let’s get back to my life-windsurf abstraction. I got carried away on the point how A and B relate. And in my windsurf-life model (WM) I claim. 
+If you are a somewhat proficient windsurfer, you will make many choices in A based on B (the absolute bearing you want to have at any point in time during your sailing session until you’re back at your van). If you are not so proficient, you will still land up at your van eventually, but maybe you will have to do the ‘walk of shame’ along the beach, carrying your gear against the wind through the sand. With 40 knots sideshore this gives you great exercise and the most natural skin peeling you can imagine. You need a lot of abs for that. Abs are not only made in the kitchen, but also on the beach.
+
+Let me recapitulate the key terms of the WM:
+- external conditions (EC)
+- internal conditions (ic)
+- target Bearing (tb)
+- Strategy (s)
+- effective bearing (EB)
```
Comment (KathSupp) [inline windsurf-life-strategy.md:83]: You should explain this.

File: windsurf-life-strategy.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+A: 
+Your gear and the way you apply it: The opening angle of the sail with respect to the board length axis, the angle between board and wind direction. Many of the internal conditions but not all.
+B: 
+your absolute bearing 
+
+Strictly speaking B is determined by A. 
+BUT: You have an infinite set of choices in A and you have to pick one. Which one are you gonna take? This brings us to the next Keyword.
+
+Strategy: No matter what you think what your strategy is, even when you think you don’t follow any strategy, there is still something I call 
+‘effective strategy (ES)’: Meaning even if you think you don’t follow any strategy, for the outside universe, it will still look like you follow one. Imagine some observer O in another universe, watching you in your life through a wormhole. What will he/she/it see and think and write in a report about you (Y)? For instance:
```
Comment (KathSupp) [inline windsurf-life-strategy.md]: Connection between ES and ‚Plan‘ (line 71)?

File: windsurf-life-strategy.md (hunk 1 above)
Comment (altrupedia) [reply windsurf-life-strategy.md:67]: Not necessary but I like it ;)

**Pull request merged**